### PR TITLE
add bootstrap

### PR DESCRIPTION
**Pull request by justinegab**
Adds bootstrap to our site.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,6 +5,8 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO"
+        crossorigin="anonymous">
   <link rel="stylesheet" href="styles/normalize.css">
```
Comment (mckalexee) [inline index.html]: If we add bootstrap, we should remove normalize.css

**Review by mckalexee — CHANGES_REQUESTED** [review]
Can we remove normalize CSS if we're adding bootstrap?

File: index.html (hunk 1 above)
Comment (justinegab) [reply index.html]: Removed normalize.css

**Review by mckalexee — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**